src/main/resources/META-INF/plugin.xml

```diff
@@ -3,22 +3,6 @@
     <name>AsyncAPI</name>
     <vendor email="pavelbodyachevskiy@gmail.com" url="https://www.asyncapi.com">AsyncAPI</vendor>
 
-    <description><![CDATA[
-    <h2>⚠️ This plugin doesn't support AsyncAPI 1.x</h2>
```

Comment:
Are we no longer stating that the plugin doesn't support AsyncAPI v1.x?

Reply:
Yep, It almost passed, 5 years ,from it's release